app/views/admin/draft_edition_change_notes/edit_change_note.html.erb

```diff
@@ -0,0 +1,50 @@
+<% content_for :page_title, "#{@edition.title}: Change note" %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @change_note_form)) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for @change_note_form, url: update_change_note_admin_edition_path, method: :patch do |form| %>
+      <%= render "govuk_publishing_components/components/radio", {
```

Comment:
this is almost a direct rip from the `_change_notes` partial. I've not bothered refactoring it into a partial as the design might change etc. but it may well be worth doing if we proceed down this route